e2e_playwright/test_assets/iframerResizer.min.js

```diff
@@ -1,3 +1,19 @@
+/**
```

Comment:
We shouldn't add our license header here. I think we need to configure this file to be ignored for the license header.